lib/service_sign_in/check-state-pension.cy.yaml

```diff
@@ -0,0 +1,49 @@
+start_page_slug: gwirio-wladwriaeth-pensiwn
+locale: cy
+choose_sign_in:
+  title: Profi pwy ydych er mwyn mynd yn eich blaen
+  slug: profi-pwy-ydych
+  options:
+    - text: Defnyddio Porth y Llywodraeth
+      url: https://www.tax.service.gov.uk/gg/sign-in?continue=/personal-account&accountType=individual&origin=PTA-frontend
+      hint_text: Bydd gennych Ddynodydd Defnyddiwr (ID) os ydych wedi cofrestru i wneud pethau fel cyflwyno'ch Ffurflen Dreth Hunanasesiad ar-lein.
+    - text: Defnyddio GOV.UK Verify
+      url: https://www.tax.service.gov.uk/personal-account/start-verify
+      hint_text: Bydd gennych gyfrif os ydych eisoes wedi profi pwy ydych gyda naill ai Barclays, CitizenSafe, Digidentity, Experian, Swyddfa'r Post, y Post Brenhinol neu SecureIdentity.
+    - text: Defnyddiwch hunaniaeth digidol o wlad Ewropeaidd arall
+      url: https://www.tax.
+      service.gov.uk/personal-account/start-verify?journey_hint=eidas_sign_in
```

Comment:
This needs to continue on the previous line

Reply:
An eagle eye! Thanks for that, fixed!